doc/classes/CanvasItem.xml

```diff
@@ -44,7 +44,7 @@
 			<argument index="6" name="width" type="float" default="1.0" />
 			<argument index="7" name="antialiased" type="bool" default="false" />
 			<description>
-				Draws an arc between the given angles. The larger the value of [code]point_count[/code], the smoother the curve.
+				Draws a non-filled arc between the given angles. The larger the value of [code]point_count[/code], the smoother the curve. See also [method draw_circle].
```

Comment:
"non-filled arc" is technically redundant, but I feel it's better to be explicit here :slightly_smiling_face:

Reply:
Doesn't really matter, but I think "unfilled" sounds more idiomatic